src/components/PieCharts/CustomActiveDonut/CustomActiveDonutView.js

```diff
@@ -127,7 +127,7 @@ class CustomActiveDonut extends PureComponent {
     const COLORS_ODD = (colors && colors.odd) ? colors.even : DEFAULT_COLORS_ODD;
 
     const { activeIndex } = this.state;
-
+    const paddingAngle = paddingSpace ? paddingSpace : 0;
```

Comment:
CustomActiveDonut.defaultProps = {
  titleText: 'Cases',
 paddingSpace: 0
};

This will do the trick, You can directly refer to the prop where ever you needed

Reply:
Updated